target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,350 @@
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <7>;
+			gpios = <&aw9523 9 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-1 { /* p1_2 - wifi12 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&aw9523 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led-5 { /* p1_3 - rgb13-green */
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <8>;
+			gpios = <&aw9523 11 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-2 { /* p1_4 - phone14 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <9>;
+			gpios = <&aw9523 12 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <50>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&button_pins>;
+
+		key-wps {
+			gpios = <&aw9523 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		key-wlan {
+			gpios = <&aw9523 15 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "Bootloader";
+			reg = <0x0 0x80000>;
+			read-only;
+		};
+
+		partition@80000 {
+			label = "Config";
+			reg = <0x80000 0x80000>;
+			read-only;
+		};
+
+		partition@100000 {
+			label = "Factory";
+			reg = <0x100000 0x100000>;
+			read-only;
+
+			nvmem-layout {
+				compatible = "fixed-layout";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				eeprom0: eeprom@0 {
+					reg = <0x0 0x400>;
+				};
+
+				eeprom1: eeprom@8000 {
+					reg = <0x8000 0x200>;
+				};
+
+				macaddr_factory_4: macaddr@4 {
+					reg = <0x4 0x6>;
+				};
+
+				macaddr_factory_e000: macaddr@e000 {
+					reg = <0xe000 0x6>;
+				};
+
+				macaddr_factory_e006: macaddr@e006 {
+					reg = <0xe006 0x6>;
+				};
+
+				macaddr_factory_8004: macaddr@8004 {
+					reg = <0x8004 0x6>;
+				};
+			};
+		};
+
+		partition@200000 {
+			label = "inventory";
+			reg = <0x200000 0x100000>;
+			read-only;
+		};
+
+		partition@300000 {
+			label = "customer";
+			reg = <0x300000 0x100000>;
+			read-only;
+		};
+
+		partition@400000 {
+			label = "zone";
+			reg = <0x400000 0x100000>;
+			read-only;
+		};
+
+		partition@500000 {
+			label = "reserved0";
+			reg = <0x500000 0x500000>;
+			read-only;
+		};
+
+		partition@a00000 {
+			label = "firmware";
+			reg = <0xa00000 0x500000>;
+		};
+
+		partition@f00000 {
+			label = "ubi";
+			reg = <0xf00000 0x2400000>;
+		};
+
+		/* factory firmware layout
+
+		partition@a00000 {
+			label = "kernel0";
+			reg = <0xa00000 0x500000>;
+		};
+
+		partition@f00000 {
+			label = "kernel1";
+			reg = <0xf00000 0x500000>;
+		};
+
+		partition@1400000 {
+			label = "ubi0";
+			reg = <0x1400000 0x2400000>;
+		};
+
+		partition@3800000 {
+			label = "ubi1";
+			reg = <0x3800000 0x2400000>;
+		};
+		 */
+
+		partition@5c00000 {
```

Comment:
Maybe those reserved partitions are usable and they can be used to extend space for user data.